src/canvas/Renderer.tsx

```diff
@@ -315,12 +360,34 @@ export class Renderer implements IRenderer {
     };
 
     private _tick = (): void => {
-        // this._update();
         this._draw();
     
         requestAnimationFrame(this._tick);
     };
 
+    private _updateCanvasSize = (first: boolean): void => {
```

Comment:
since this is only used at init, rename to "_initCanvasSize" and remove argument